### PR TITLE
chili/chilly peppers sprite alignment fix & minor refactorization

### LINKED ISSUE #25019: Cold chili pepper sprite is off-centered
## Description

When you try to place it on the table, it ends up placing weirdly and it just feels odd to use anyway because of its bottom-right sprite alignment.

**Screenshots**

image

**Comparison to normal chilli pepper:**

image

### PR DESCRIPTION
**Pull request by Gyrandola**
## About the PR

Chilly pepper sprite was way off-centered, turns out the normal chili sprite wasn't perfectly aligned as well.
Refactored the names and IDs to make it clear they're both peppers.

## Why / Balance

Fixes #25019.
New naming is clearer.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Screenshot 2024-03-10 171917

## Breaking changes

**Changelog**

:cl:
- fix: Pepper sprites are now properly centered!

### CODE REVIEW

**Comment by potato1234x** [comment]
name change is kinda unnecessary its pretty obvious theyre peppers

**Comment by Luxzhv** [comment]
> name change is kinda unnecessary its pretty obvious theyre peppers

Better to just have it, to also be ready for Bell Peppers and Pepper corns whenever someone gets around to it.

**Comment by Tayrtahn** [comment]
I've seen "chili" in the files and assumed it was the stew, so I agree that the change is good.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -1106,9 +1106,9 @@
           Quantity: 1
 
 - type: entity
-  name: chili
+  name: chili pepper
   parent: FoodProduceBase
-  id: FoodChili
+  id: FoodChiliPepper
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:1111]: you need to add entries to migrations.yml for both of these prototypes.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

**Pull request merged**